extensions/blocks/conversation/index.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import { __, _x } from '@wordpress/i18n';
+
+/**
+ * External dependencies
+ */
+import { ConversationIcon as icon } from '../../shared/icons';
+
+/**
+ * Local dependencies
+ */
+import attributes from './attributes';
+import edit from './edit';
+import save from './save';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+export const name = 'conversation';
+export const title = __( 'Conversation', 'jetpack' );
+export const settings = {
+	title,
+	description: __( 'Conversation', 'jetpack' ),
```

Comment:
Maybe we should expand on this a little bit, to explain what the block can be used for?